lib/computer_opponent.rb

```diff
@@ -0,0 +1,7 @@
+class ComputerOpponent
+
+  def computer_choice
+    ["Rock", "Paper", "Scissors"].sample
+  end
+
+end
```

Comment:
I like that you have pulled the npc into its own class, shows clear use of the single responsibility principle